File: Project/Client/MyApplicationProject/MyApplication/src/main/java/com/example/myapplication/dtos/FriendDeletionDTO.java
package com.example.myapplication.dtos;

/**
 * DTO object used when deleting a friend from user's list of friends.
 */
public class FriendDeletionDTO
{
    private int UserId;
    private int FriendId;

    public FriendDeletionDTO(int userId, int friendId) {
        UserId = userId;
        FriendId = friendId;
    }
}

File: Project/WebService/Services/DTOs/FriendDeletionDTO.cs
// --------------------------------------------------------------------------------------------------------------------
// <copyright file="FriendDeletionDTO.cs" company="">
//   
// </copyright>
// <summary>
//   The friend deletion dto.
// </summary>
// --------------------------------------------------------------------------------------------------------------------

namespace Services.DTOs
{
    using System.Runtime.Serialization;

    /// <summary>
    /// The friend deletion dto.
    /// </summary>
    [DataContract]
    public class FriendDeletionDTO
    {
        /// <summary>
        /// The user id.
        /// </summary>
        [DataMember]
        public int UserId;

        /// <summary>
        /// The friend id.
        /// </summary>
        [DataMember]
        public int FriendId;
    }
}